### PR TITLE
close #340

### LINKED ISSUE #340: remove `datawizard` functions
https://github.com/easystats/datawizard/issues/3

- [x] adjust
- [x] change_scale
- [x] normalize
- [x] ranktransform (make sure to import!)
- [x] standardize (but not the default method! Make sure to import)
- [x] unstandardize

### PR DESCRIPTION
**Pull request by mattansb**

### CODE REVIEW

File: R/standardize.models.R [hunk 1]
```diff
@@ -1,16 +1,33 @@
-#' @rdname standardize
+#' Re-fit a model with standardized data
+#'
+#' Performs a standardization of data (z-scoring) using
+#' [`datawizrd:standardize()`] and then re-fits the model to the standardized
```
Comment (IndrajeetPatil) [inline R/standardize.models.R:4]: Typo: `[`datawizard:standardize()`]`

Comment (mattansb) [reply R/standardize.models.R:4]: Ah, there it is! Thanks!

**Comment by mattansb** [comment]
@IndrajeetPatil do you want to copy the "data standardization" vignette that use to be here?

**Comment by IndrajeetPatil** [comment]
Thanks for pointing this out!

I've added it to `datawizard` now:
https://github.com/easystats/datawizard/commit/25abc9fe1aa2239cc00ac39bc2497cf0da0fb9b6

**Comment by bwiernik** [comment]
One thing we might consider is offloading the vignettes just to the pkgdown sites, like the tidyverse packages do (eg, https://dplyr.tidyverse.org/articles/index.html). We could include a table of contents vignette in the packages themselves perhaps (and we could cross-link across packages as needed)

**Comment by mattansb** [comment]
This has been brought up before - so long as CRAN doesn't give us trouble, we'll keep them in (:

**Pull request merged**